config.yaml

```
@@ -325,6 +328,46 @@ orgs:
         privacy: closed
         repos:
           ansible-snapshot: admin
+      ansible-zipship:
+        description: "Ansible ZipShip"
+        maintainers:
+          - A-Beck
+          - sabre1041
+          - bontreger
+          - brady-spiva
+          - jeffcpu
```

Maybe this one too @sabre1041? Sorry I missed it before
```suggestion
          - jeffcpullen
```